MovieQuiz/Services/NetworkClient.swift

```diff
@@ -0,0 +1,43 @@
+//
+//  NetworkClient.swift
+//  MovieQuiz
+//
+//  Created by Marina on 28.10.2022.
+//
+
+import Foundation
+
+/// Отвечает за загрузку данных по URL
+ struct NetworkClient {
+
+    private enum NetworkError: Error {
+        case codeError
+    }
+
+// Делает запрос в интернет и возвращает результат
+// Хэндлер принимает на вход объект "результат", который принимает значение либо "данные", либо "ошибка"
```

Comment:
Можно у названий функций оставлять так называемый doc-коммент. Тогда Xcode будет показывать информацию о методе если на него нажать с cmd 
```suggestion
    /// Делает запрос в интернет и возвращает результат
    /// Хэндлер принимает на вход объект "результат", который принимает значение либо "данные", либо "ошибка"
```